docs/topics/strings.md

```diff
@@ -158,9 +158,13 @@ To format a string to your specific requirements, use the [`String.format()`](ht
 function. 
 
 The `String.format()` function accepts a format string and one or more arguments. The format string contains one placeholder 
-(`%`) for each remaining argument, followed by [format specifiers](https://docs.oracle.com/javase/8/docs/api/java/util/Formatter.html#summary). 
-Format specifiers are formatting instructions for the respective argument. In the output, each argument fills its 
-corresponding placeholder in the defined format:
+(`%`) for each remaining argument, followed by format specifiers. 
```

Comment:
While we're clarifying it, I would add that it's _usually_ one placeholder for each argument.
Because with the advanced syntax (`argument_index$`), one can provide more than one placeholder that refer to a single argument. Some examples of that are in https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/format.html

Reply:
That's an excellent idea for an addition! I added an additional line to describe it and modified the code example to reference the description.